### DISCUSSION TITLE
A Problem

### DISCUSSION THREAD

**Discussion by DjRafael50** [AzuraCast/AzuraCast#7508] [General]
I'm facing a problem when changing tracks, when I'm finishing a track it freezes and returns, "it seems to choke" I solved it by restarting all the stations, after a few days the problem returns, I have to restart everything disconnecting the listeners. Does anyone know how to resolve this? Thanks

**Comment by Heimatradio** [comment]
Unfortunately, that is not enough information about your server and its configuration.
I suspect that the server is overloaded. I have personally experienced this with many providers.
Unfortunately, many are not suitable for AzuraCast at all.
I have changed servers more than 10 times, and now I have finally found a good one that runs without problems and without overload.

**Reply by DjRafael50** [reply]
Hello, I use Hetzner, my configuration is 8/16 CPU and 16GB of RAM, the processor never exceeds 15% of the CPU. I don't know why this happens, I've looked at the logs but I don't see anything. I restarted the stations and in about 3 to 4 days the problem starts when cueing between tracks, I don't know how to explain it. Anyway, I'll try to get help, thanks for your attention.

**Reply by bombelman** [reply]
> I have changed servers more than 10 times, and now I have finally found a good one that runs without problems and without overload.

Curious to know which is the good one ;-)

**Reply by Heimatradio** [reply]
Hetzner, yes i use the webserver that's running good, the vps weren't good there either. 

Curious to know which is the good one ;-)
Strato.de , now i use the VPS Linux VC6-12
since then, all errors have disappeared
have also a look here https://github.com/AzuraCast/AzuraCast/discussions/7462

**Comment by unudelaradio** [comment]
> I restarted the stations and in about 3 to 4 days the problem starts when cueing between tracks, I don't know how to explain it. 

@DjRafael50 This also happens to me, mate! I know perfectly what you are talking about. And is happening since older versions, no matter the VPS provider. I'm surprised that not so many users talk about this issue. 

I also mentioned this problem, recently, here: https://github.com/AzuraCast/AzuraCast/discussions/7475#discussioncomment-10791174.

I hope that future Liquidsoap and Azuracast versions will come with changes, solving these choke/glitches at the end of tracks, before cueing starts. As you said, for the moment, the only way to solve it is restarting stations.

**Reply by DjRafael50** [reply]
Thanks, I thought it might be something I did wrong in the config or installation

**Comment by unudelaradio** [comment]
I think this is more like a buffering problem. On one of my stations, that hasn't been restarted for a month now, the behaviour is even critical, accentuate. Before cueing starts the pauses are even longer, for seconds, causing sometimes "reconnection" in AIMP Player. But no matter the player, or browser, the pauses are there. Long pauses, after each track, right before mixing!

I have set and checked some audio recordings in Liquidsoap. There's no problem, no pauses, no glitches. So this issue can be spotted only when listening to the stream. Appears after 3 or 4 days after the mount point starts, and accentuates day after day.

Once again... here's how it sounds: https://drive.google.com/file/d/1mjDyB-PRaBdVZANoEuVzWdIjOtPME0H7/
